### ISSUE TITLE
Upload issue

### ISSUE THREAD

**Issue by isoos** [dart-lang/pub-dartlang-dart#1680]
@jodinathan reported on gitter:

````
Uploading... (36.6s)
It looks like pub.dartlang.org is having some trouble.
Pub will wait for a while before trying to connect again.
HTTP error 500: Internal Server Error at https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=d6aaa85d-a597-4043-a4fe-fda748f69d1e&bucket=pub-packages&key=tmp/d6aaa85d-a597-4043-a4fe-fda748f69d1e&etag=3dea35daa2922ddc17a8527ce185ff08.
This is likely a transient error. Please try again later.
````

Related stack trace:

````
pub.cloud_repository: An error occured while streaming tarball to FS.

Error:
    DetailedApiRequestError(status: 404, message: No error details. HTTP status was: 404.)

Stack:
    #0      _validateResponse.throwGeneralError (package:_discoveryapis_commons/src/clients.dart:862:7)
    #1      _validateResponse (package:_discoveryapis_commons/src/clients.dart:888:24)
    #2      StackZoneSpecification._registerUnaryCallback.<anonymous closure>.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:26)
    #3      StackZoneSpecification._run (package:stack_trace/src/stack_zone_specification.dart:209:15)
    #4      StackZoneSpecification._registerUnaryCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:14)
    #5      _rootRunUnary (dart:async/zone.dart:1132:38)
    #6      _CustomZone.runUnary (dart:async/zone.dart:1029:19)
    #7      _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
    #8      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
    #9      Future._propagateToListeners (dart:async/future_impl.dart:671:32)
    #10     Future._complete (dart:async/future_impl.dart:476:7)
    #11     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
    #12     StackZoneSpecification._registerUnaryCallback.<anonymous closure>.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:26)
    #13     StackZoneSpecification._run (package:stack_trace/src/stack_zone_specification.dart:209:15)
    #14     StackZoneSpecification._registerUnaryCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:14)
    #15     _rootRunUnary (dart:async/zone.dart:1132:38)
    #16     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
    #17     _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
    #18     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
    #19     Future._propagateToListeners (dart:async/future_impl.dart:671:32)
    #20     Future._complete (dart:async/future_impl.dart:476:7)
    #21     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
    #22     _AsyncAwaitCompleter.complete (dart:async/runtime/libasync_patch.dart:28:18)
    #23     _completeOnAsyncReturn (dart:async/runtime/libasync_patch.dart:295:13)
    #24     AuthenticatedClient.send (package:googleapis_auth/src/auth_http_utils.dart)
    <asynchronous suspension>
    #25     _MetadataServerClient.send (package:googleapis_auth/auth_io.dart:349:25)
    <asynchronous suspension>
    #26     ApiRequester._request.simpleRequest (package:_discoveryapis_commons/src/clients.dart:221:26)
    #27     ApiRequester._request (package:_discoveryapis_commons/src/clients.dart:250:25)
    #28     ApiRequester.request (package:_discoveryapis_commons/src/clients.dart:66:12)
    #29     ObjectsResourceApi.get (package:googleapis/storage/v1.dart:2832:32)
    #30     _BucketImpl.read (file:///root/.pub-cache/hosted/pub.dartlang.org/gcloud-0.6.0+1/lib/src/storage_impl.dart:224:10)
    <asynchronous suspension>
    #31     TarballStorage.readTempObject (package:pub_dartlang_org/frontend/backend.dart:783:14)
    #32     GCloudPackageRepository.finishAsyncUpload.<anonymous closure>.<anonymous closure> (package:pub_dartlang_org/frontend/backend.dart:289:40)
    <asynchronous suspension>
    #33     withTempDirectory.<anonymous closure> (package:pub_dartlang_org/shared/utils.dart:46:16)
    #34     StackZoneSpecification._registerUnaryCallback.<anonymous closure>.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:26)
    #35     StackZoneSpecification._run (package:stack_trace/src/stack_zone_specification.dart:209:15)
    #36     StackZoneSpecification._registerUnaryCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:129:14)
    #37     _rootRunUnary (dart:async/zone.dart:1132:38)
    #38     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
    #39     _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
    #40     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
    #41     Future._propagateToListeners (dart:async/future_impl.dart:671:32)
    #42     Future._completeWithValue (dart:async/future_impl.dart:486:5)
    #43     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:516:7)
    #44     StackZoneSpecification._run (package:stack_trace/src/stack_zone_specification.dart:209:15)
    #45     StackZoneSpecification._registerCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart:119:48)
    #46     _rootRun (dart:async/zone.dart:1124:13)
    #47     _CustomZone.run (dart:async/zone.dart:1021:19)
    #48     _CustomZone.runGuarded (dart:async/zone.dart:923:7)
    #49     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:963:23)
    #50     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
    #51     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
    #52     _runPendingImmediateCallback (dart:isolate/runtime/libisolate_patch.dart:115:13)
    #53     _RawReceivePortImpl._handleMessage (dart:isolate/runtime/libisolate_patch.dart:172:5)
````

**Comment by isoos** [comment]
@jodinathan: could you please give a bit more detail: what is the project size? Is it a flutter plugin or a regular dart package?

**Comment by jodinathan** [comment]
it is a regular dart package. 
Basically an interface to a InputMask JS lib.

https://github.com/jodinathan/input_mask

**Comment by isoos** [comment]
I've just uploaded a similar-sized package (a new version for an already existing package).
@jodinathan: Could you please include the log with `pub publish -v` (verbose logging)?

**Comment by jodinathan** [comment]
```
MBP-de-Jonathan:input_mask jonathanrezende$ pub publish -v
FINE: Pub 2.0.0
IO  : Spawning "git --version" in /Users/jonathanrezende/Dropbox/Projects/input_mask/.
IO  : Finished git. Exit code 0.
    | stdout:
    | | git version 2.17.1 (Apple Git-112)
    | Nothing output on stderr.
FINE: Determined git command null.
IO  : Spawning "git check-ignore --quiet ." in /Users/jonathanrezende/Dropbox/Projects/input_mask/.
IO  : Finished git. Exit code 1.
    | Nothing output on stdout.
    | Nothing output on stderr.
IO  : Spawning "git ls-files --cached --others --exclude-standard" in /Users/jonathanrezende/Dropbox/Projects/input_mask/.
IO  : Finished git. Exit code 0.
    | stdout:
    | | .gitignore
    | | CHANGELOG.md
    | | LICENSE
    | | README.md
    | | analysis_options.yaml
    | | example/input_mask_example.dart
    | | lib/input_mask.dart
    | | lib/src/interface.dart
    | | pubspec.yaml
    | | test/input_mask_test.dart
    | Nothing output on stderr.
FINE: Archiving and publishing input_mask 0.0.1 (.).
MSG : Publishing input_mask 0.0.1 to https://pub.dartlang.org:
    | |-- .gitignore
    | |-- CHANGELOG.md
    | |-- LICENSE
    | |-- README.md
    | |-- analysis_options.yaml
    | |-- example
    | |   '-- input_mask_example.dart
    | |-- lib
    | |   |-- input_mask.dart
    | |   '-- src
    | |       '-- interface.dart
    | |-- pubspec.yaml
    | '-- test
    |     '-- input_mask_test.dart
FINE: Creating .tar.gz stream containing:
    | ./.gitignore
    | ./CHANGELOG.md
    | ./LICENSE
    | ./README.md
    | ./analysis_options.yaml
    | ./example/input_mask_example.dart
    | ./lib/input_mask.dart
    | ./lib/src/interface.dart
    | ./pubspec.yaml
    | ./test/input_mask_test.dart
IO  : Spawning "git ls-files --cached --others --exclude-standard" in /Users/jonathanrezende/Dropbox/Projects/input_mask/.
IO  : Finished git. Exit code 0.
    | stdout:
    | | .gitignore
    | | CHANGELOG.md
    | | LICENSE
    | | README.md
    | | analysis_options.yaml
    | | example/input_mask_example.dart
    | | lib/input_mask.dart
    | | lib/src/interface.dart
    | | pubspec.yaml
    | | test/input_mask_test.dart
    | Nothing output on stderr.
IO  : Spawning "git ls-files --cached --others --exclude-standard" in /Users/jonathanrezende/Dropbox/Projects/input_mask/.
IO  : Finished git. Exit code 0.
    | stdout:
    | | .gitignore
    | | CHANGELOG.md
    | | LICENSE
    | | README.md
    | | analysis_options.yaml
    | | example/input_mask_example.dart
    | | lib/input_mask.dart
    | | lib/src/interface.dart
    | | pubspec.yaml
    | | test/input_mask_test.dart
    | Nothing output on stderr.
IO  : Spawning "git ls-files --cached --others --exclude-standard" in /Users/jonathanrezende/Dropbox/Projects/input_mask/.
IO  : Finished git. Exit code 0.
    | stdout:
    | | .gitignore
    | | CHANGELOG.md
    | | LICENSE
    | | README.md
    | | analysis_options.yaml
    | | example/input_mask_example.dart
    | | lib/input_mask.dart
    | | lib/src/interface.dart
    | | pubspec.yaml
    | | test/input_mask_test.dart
    | Nothing output on stderr.
IO  : Spawning "git ls-files --cached --others --exclude-standard" in /Users/jonathanrezende/Dropbox/Projects/input_mask/.
IO  : Finished git. Exit code 0.
    | stdout:
    | | .gitignore
    | | CHANGELOG.md
    | | LICENSE
    | | README.md
    | | analysis_options.yaml
    | | example/input_mask_example.dart
    | | lib/input_mask.dart
    | | lib/src/interface.dart
    | | pubspec.yaml
    | | test/input_mask_test.dart
    | Nothing output on stderr.
IO  : Reading binary file ./README.md.
IO  : Read 947 bytes from ./README.md.
IO  : Spawning "tar --format=ustar --create --gzip --directory /Users/jonathanrezende/Dropbox/Projects/input_mask/. @/dev/stdin" in /Users/jonathanrezende/Dropbox/Projects/input_mask/.
WARN: Suggestions:
WARN: * line 5, column 1 of lib/src/interface.dart: This package doesn't depend on js.
    |   import 'package:js/js.dart';
    |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FINE: Showing confirm message: 
    | Package has 1 warning. Upload anyway

Package has 1 warning. Upload anyway (y/n)? y
FINE: Loading OAuth2 credentials.
FINE: Saving OAuth2 credentials.
IO  : Writing 400 characters to text file /Users/jonathanrezende/.pub-cache/credentials.json.
MSG : Uploading...
IO  : HTTP GET https://pub.dartlang.org/api/packages/versions/new
    | Accept: application/vnd.pub.v2+json
    | authorization: <censored>
    | user-agent: Dart pub 2.0.0
IO  : HTTP response 200 OK for GET https://pub.dartlang.org/api/packages/versions/new
    | took 0:00:02.129904
    | transfer-encoding: chunked
    | date: Mon, 08 Oct 2018 18:30:18 GMT
    | content-encoding: gzip
    | vary: Accept-Encoding
    | via: 1.1 google
    | content-type: application/json
    | x-frame-options: SAMEORIGIN
    | x-xss-protection: 1; mode=block
    | x-content-type-options: nosniff
    | server: dart:io with Shelf
IO  : HTTP POST https://storage.googleapis.com
    | authorization: <censored>
    | content-type: multipart/form-data; boundary=dart-http-boundary-D.JH.58ZLbA78RMJRzaDpQR4EjVPkXaAHG1x7mpU2s78bC95_hm
    | user-agent: Dart pub 2.0.0
IO  : HTTP response 303 See Other for POST https://storage.googleapis.com
    | took 0:00:03.868519
    | alt-svc: quic=":443"; ma=2592000; v="44,43,39,35"
    | location: https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | date: Mon, 08 Oct 2018 18:30:23 GMT
    | vary: Origin
    | content-length: 0
    | x-guploader-uploadid: AEnB2UqTmTTWF-m2pX350N4-HjORu07y4KEyTTqqdT-7uh-W6LCWZF_2OasQ3s3phyKzcur-w1cEIWm7xlllicD0ALfCTRMcOg
    | content-type: text/html; charset=UTF-8
    | server: UploadServer
IO  : HTTP GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | Accept: application/vnd.pub.v2+json
    | authorization: <censored>
    | user-agent: Dart pub 2.0.0
IO  : HTTP response 500 Internal Server Error for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | took 0:00:00.765156
    | date: Mon, 08 Oct 2018 18:30:24 GMT
    | content-length: 91
    | via: 1.1 google
    | content-type: application/json
    | x-frame-options: SAMEORIGIN
    | x-xss-protection: 1; mode=block
    | x-content-type-options: nosniff
    | server: dart:io with Shelf
IO  : Retry #1 for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9...
IO  : HTTP GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | Accept: application/vnd.pub.v2+json
    | authorization: <censored>
    | user-agent: Dart pub 2.0.0
IO  : HTTP response 500 Internal Server Error for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | took 0:00:00.412197
    | date: Mon, 08 Oct 2018 18:30:24 GMT
    | content-length: 111
    | via: 1.1 google
    | content-type: application/json
    | x-frame-options: SAMEORIGIN
    | x-xss-protection: 1; mode=block
    | x-content-type-options: nosniff
    | server: dart:io with Shelf
IO  : Retry #2 for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9...
IO  : HTTP GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | Accept: application/vnd.pub.v2+json
    | authorization: <censored>
    | user-agent: Dart pub 2.0.0
IO  : HTTP response 500 Internal Server Error for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | took 0:00:00.441513
    | date: Mon, 08 Oct 2018 18:30:26 GMT
    | content-length: 111
    | via: 1.1 google
    | content-type: application/json
    | x-frame-options: SAMEORIGIN
    | x-xss-protection: 1; mode=block
    | x-content-type-options: nosniff
    | server: dart:io with Shelf
IO  : Retry #3 for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9...
IO  : HTTP GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | Accept: application/vnd.pub.v2+json
    | authorization: <censored>
    | user-agent: Dart pub 2.0.0
IO  : HTTP response 500 Internal Server Error for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | took 0:00:00.390845
    | date: Mon, 08 Oct 2018 18:30:28 GMT
    | content-length: 111
    | via: 1.1 google
    | content-type: application/json
    | x-frame-options: SAMEORIGIN
    | x-xss-protection: 1; mode=block
    | x-content-type-options: nosniff
    | server: dart:io with Shelf
IO  : Retry #4 for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9...
MSG : It looks like pub.dartlang.org is having some trouble.
    | Pub will wait for a while before trying to connect again.
IO  : HTTP GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | Accept: application/vnd.pub.v2+json
    | authorization: <censored>
    | user-agent: Dart pub 2.0.0
IO  : HTTP response 500 Internal Server Error for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | took 0:00:00.573225
    | date: Mon, 08 Oct 2018 18:30:58 GMT
    | content-length: 111
    | via: 1.1 google
    | content-type: application/json
    | x-frame-options: SAMEORIGIN
    | x-xss-protection: 1; mode=block
    | x-content-type-options: nosniff
    | server: dart:io with Shelf
IO  : Retry #5 for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9...
IO  : HTTP GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | Accept: application/vnd.pub.v2+json
    | authorization: <censored>
    | user-agent: Dart pub 2.0.0
IO  : HTTP response 500 Internal Server Error for GET https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9
    | took 0:00:00.576774
    | date: Mon, 08 Oct 2018 18:31:29 GMT
    | content-length: 111
    | via: 1.1 google
    | content-type: application/json
    | x-frame-options: SAMEORIGIN
    | x-xss-protection: 1; mode=block
    | x-content-type-options: nosniff
    | server: dart:io with Shelf
FINE: Uploading finished (1:12.126s).
FINE: Saving OAuth2 credentials.
IO  : Writing 400 characters to text file /Users/jonathanrezende/.pub-cache/credentials.json.
ERR : HTTP error 500: Internal Server Error at https://pub.dartlang.org/api/packages/versions/newUploadFinish?upload_id=0c6d1d01-3804-4415-99bd-a45dec18d858&bucket=pub-packages&key=tmp/0c6d1d01-3804-4415-99bd-a45dec18d858&etag=97d1f9140b4f391d936fa28a4fa1d9c9.
    | This is likely a transient error. Please try again later.
FINE: Exception type: ApplicationException
FINE: package:pub/src/utils.dart 573:5                fail
    | package:pub/src/http.dart 216:7                 _ThrowingClient.send
    | ===== asynchronous gap ===========================
    | dart:async                                      _AsyncAwaitCompleter.completeError
    | package:pub/src/http.dart                       _ThrowingClient.send
    | ===== asynchronous gap ===========================
    | dart:async                                      _asyncThenWrapperHelper
    | package:pub/src/http.dart                       _ThrowingClient.send
    | package:http_throttle/http_throttle.dart 33:31  ThrottleClient.send
    | ===== asynchronous gap ===========================
    | dart:async                                      _asyncThenWrapperHelper
    | package:http_throttle/http_throttle.dart        ThrottleClient.send
    | package:oauth2/src/client.dart 109:38           Client.send
    | ===== asynchronous gap ===========================
    | dart:async                                      _asyncThenWrapperHelper
    | package:pub/src/command/lish.dart               LishCommand._publish.<fn>.<fn>
    | package:pub/src/log.dart 378:18                 progress
    | package:pub/src/command/lish.dart 68:16         LishCommand._publish.<fn>
    | package:pub/src/oauth2.dart 84:14               withClient.<fn>
    | ===== asynchronous gap ===========================
    | dart:async                                      Future.then
    | package:pub/src/oauth2.dart 83:28               withClient
    | package:pub/src/command/lish.dart 67:13         LishCommand._publish
    | dart:async                                      _AsyncAwaitCompleter.start
    | package:pub/src/command/lish.dart 64:18         LishCommand._publish
    | package:pub/src/command/lish.dart 145:13        LishCommand.run
    | ===== asynchronous gap ===========================
    | dart:async                                      _asyncThenWrapperHelper
    | package:pub/src/command/lish.dart               LishCommand.run
    | package:args/command_runner.dart 194:27         CommandRunner.runCommand
MBP-de-Jonathan:input_mask jonathanrezende$ 

```

**Comment by jodinathan** [comment]
added the JS dependency ```js: ^0.6.1+1``` to the pubspec.yaml but no success

**Comment by isoos** [comment]
@sortie: any idea what could be happening here? It looks like that reading the uploaded temp object from the storage fails. Could you please check the prod data if there is an object there?

**Comment by jodinathan** [comment]
any news on this?
It is still not possible to publish the package.
if anything I could help on

**Comment by sortie** [comment]
Thank for for your patience. We did some digging and ruled out some
possibilities (your uploaded tarball didn't get stuck in our temporary
directory), but we didn't find an explanation just yet. We'll continue to
look into what's going on.

**Comment by sortie** [comment]
@isoos What's the next step here? Let me know what to look for in the production data?

**Comment by jodinathan** [comment]
it is still happening and I can't publish the package.
Can I do it manually by pub website or sort of?

**Comment by sortie** [comment]
This is just a hunch (we're still debugging, but maybe this may work for you):

It might be that your package was rejected. Check if you have e.g. set the homepage in your pubspec.yaml to a valid site (e.g. not example.com nor pub's website) and other possibly flaws (the pub client may tell you). You'll want to update your pubspec version to something you haven't attempted to upload before (I suspect a given package version may get stuck). Can you let us know if that works?

**Comment by isoos** [comment]
I'm closing this now, tracking the fix of the bug on #1717.

**Comment by meetqy** [comment]
https://www.cnblogs.com/meetqy/p/11250024.html
